src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
The HTML tag is not properly closed. It should be `<html>` instead of `<html /++2++/>`.